app/konnect/dev-portal/applications/dynamic-client-registration/okta.md

```diff
@@ -93,3 +93,15 @@ From the **My Apps** page in the Dev Portal, follow these instructions:
 After your application has been created, you will see the **Client ID** and **Client Secret**. Please store these values, they will only be shown once. Click **Proceed** to continue to the application's details page.
 
 4. Once your application is created, you will see it in Okta. From your Okta organization select **Applications** from the menu. You will see your application that was created in the Dev Portal, and its corresponding Client ID.
+
+## How to make successful request
+
+In previous steps we obtained the **Client ID** and **Client Secret**. To authorize the request we need to properly attach this client secret pair in the header. Using Insomnia it would look like this.
+
+![Insomnia DCR usage example](/assets/images/docs/konnect/dev-portal/insomnia_dcr_example.png)
+
+We can also do it using command line
+
+```sh
+curl <address_to_your_running_runtime_instance>/<route_you_registed> -H "Authorization: Basic <client_id>:<client_secret>"
+```
```

Comment:
I'd add "Where `example.com` is the address of the runtime instance you are running." after this so it's clear what example.com means in this context.